cmd/kaspawallet/config.go

```diff
@@ -54,13 +54,15 @@ type sendConfig struct {
 	Password      string  `long:"password" short:"p" description:"Wallet password"`
 	DaemonAddress string  `long:"daemonaddress" short:"d" description:"Wallet daemon server to connect to (default: localhost:8082)"`
 	ToAddress     string  `long:"to-address" short:"t" description:"The public address to send Kaspa to" required:"true"`
+	FromAddress   string  `long:"from-address" short:"a" description:"Specific public address to send Kaspa from" required:"false"`
```

Comment:
If it's not a big deal, I would prefer to directly implement it with multiple addresses (`FromAddresses`)